docs/configuration.md

```diff
@@ -209,10 +211,10 @@ Apart from these, the following properties are also available, and may be useful
   <td>(none)</td>
   <td>
     Set a special library path to use when launching the driver JVM.
-    
+
```

Comment:
I would turn it off. The whitespace itself doesn't matter at all functionally in most contexts. But it can create more noise in the diff. More important it can create spurious merge conflicts. An instance here or there doesn't matter much but I actively try to not make whitespace changes for this reason.